src/App.js

```diff
@@ -11,9 +11,8 @@ class App extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = { map_center : [40.762295, -73.968148], heatmap: false,
-                  mask_list : [], maskhole_list : [], nomask_list : [],
-                  glasses_im: 1, maskhole_im: 0, nomask_im: 0};
+    this.state = { map_center : [40.742997028, -73.96749613],
```

Comment:
Are these NYC coordinates? Let's name them in global variables. Something like

const nycLat = 40.742997028
const nycLong = -73.96749613